playwright-e2e/dss/component/input.ts

```diff
@@ -79,4 +79,25 @@ export default class Input extends WidgetBase {
       await expect(this.page.locator('button:visible', { hasText: 'Saving' })).toBeHidden();
     }
   }
+
+  public async fillSimple(value: string): Promise<void> {
+    await this.toLocator().fill(value);
+    await this.toLocator().blur();
+  }
+
+  public get currentValue(): Promise<string> {
+    return this.toLocator().inputValue();
+  }
+
+  public async blur(): Promise<void> {
+    await this.toLocator().blur();
+  }
+
+  public async focus(): Promise<void> {
+    await this.toLocator().focus();
+  }
+
+  public get maxLength(): Promise<string | null> {
+    return this.toLocator().getAttribute('maxlength');
+  }
```

Comment:
`async` and `get` modifier are incompatible. It should be `public async maxLength(): Promise<string | null>`. If you don't use `async` modifier, eslint won't find error when calling functions not using `await`.

Reply:
Actually, IDE can detect if you didn't use await, like so:

![Screenshot 2023-09-14 at 13 09 30](https://github.com/broadinstitute/ddp-angular/assets/77500504/d38224d0-c6c6-4aab-98f0-5d879f03eebb)